Casks/itch.rb

```diff
@@ -1,16 +1,16 @@
 cask "itch" do
-  version "25.5.1"
+  version "25.0.0"
   sha256 :no_check
 
   url "https://broth.itch.ovh/install-itch/darwin-amd64/LATEST/archive/default",
       verified: "broth.itch.ovh/"
-  name "itch.io"
+  name "itch"
   desc "Game client for itch.io"
   homepage "https://itch.io/app"
 
   livecheck do
-    url "https://broth.itch.ovh/itch/darwin-amd64"
-    regex(/(\d+(?:\.\d+)+)/i)
+    url :url
+    strategy :extract_plist
```

Comment:
```suggestion
    url "https://broth.itch.ovh/itch/darwin-amd64"
    regex(%r{href=.*?/v?(\d+(?:\.\d+)+)/?["' >]}i)
```

This reinstates the existing `livecheck` block while tightening up the regex a bit.